src/domain/notification/create/feature.ts

```diff
@@ -1,20 +1,19 @@
 
 import logger from '^/integrations/logging/module';
 
-import type { Requester } from '^/domain/authentication/types';
 
 import type { Types } from '../definitions';
 import createData from './createData';
 import insertData from './insertData';
 
-export default async function feature(requester: Requester, type: Types, receiverId: string, postId?: string, reactionId?: string): Promise<void>
+export default async function feature(creatorId: string, type: Types, receiverId: string, postId?: string, reactionId?: string): Promise<void>
```

Comment:
```suggestion
export default async function feature(type: Types, senderId: string, receiverId: string, postId?: string, reactionId?: string): Promise<void>
```

Since this feature is not directly bound to the requester anymore, we can start using `senderId` because its more related to the concept. Also, we can reorder the arguments to match the data model.